specification/purview/data-plane/Azure.Analytics.Purview.Scanning/stable/2023-09-01/scanningService.json

```diff
@@ -0,0 +1,9531 @@
+        }
+      ],
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/DataSourceType"
+        }
+      },
+      "discriminator": "kind"
+    },
+    "ScanRulesetList": {
+      "description": "List of scan rulesets.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of scan rulesets.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ScanRuleset"
+          }
+        },
+        "nextLink": {
+          "description": "The link to the next page of results, if any remaining results exist.",
+          "type": "string"
+        },
+        "count": {
+          "description": "The number of scan rulesets.",
+          "format": "int64",
+          "type": "integer"
+        }
+      }
+    },
+    "SystemScanRuleset": {
+      "description": "The system scan ruleset.",
+      "required": [
+        "kind"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/VersionedScanRuleset"
+        }
+      ],
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/DataSourceType"
+        }
+      },
+      "discriminator": "kind"
+    },
+    "SystemScanRulesetList": {
+      "description": "List of system scan rulesets.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of system scan rulesets.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SystemScanRuleset"
+          }
+        },
+        "nextLink": {
+          "description": "The link to the next page of results, if any remaining results exist.",
+          "type": "string"
+        },
+        "count": {
+          "description": "The number of system scan rulesets.",
+          "format": "int64",
+          "type": "integer"
+        }
+      }
+    },
+    "RecurrenceScheduleOccurrence": {
+      "description": "The occurrence of recurrence schedule.",
+      "type": "object",
+      "properties": {
+        "additionalProperties": {
+          "type": "object"
+        },
+        "day": {
+          "enum": [
+            "Sunday",
+            "Monday",
+            "Tuesday",
+            "Wednesday",
+            "Thursday",
+            "Friday",
+            "Saturday"
+          ],
+          "type": "string",
+          "description": "The day of recurrence schedule occurrence.",
+          "x-ms-enum": {
+            "name": "DayOfWeek",
+            "modelAsString": true
+          }
+        },
+        "occurrence": {
+          "format": "int32",
+          "type": "integer"
+        }
+      }
+    },
+    "RecurrenceSchedule": {
+      "description": "Schedule of recurrence.",
+      "type": "object",
+      "properties": {
+        "additionalProperties": {
+          "type": "object"
+        },
```

Comment:
As above, please explain what this field means / how it is used.

Reply:
Just removed this additionalProperties object. It was a legacy dictionary object in the RecurrenceSchedule object that shouldn't exist anymore.